middlewares/authenticator.go

```diff
@@ -88,6 +111,14 @@ func parserDigestUsers(users types.Users) (map[string]string, error) {
 	return userMap, nil
 }
 
+func getUserStringsFromFile(filename string) ([]string, error) {
```

Comment:
Since this function is just generically loading a file into memory and splitting the content by newlines, should we rather call it something like `getLinesFromFile`?